test/vars/MavenSpec.groovy

```diff
@@ -101,4 +101,14 @@ class MavenSpec extends JenkinsPipelineSpecification {
         then:
         1 * getPipelineMock("sh")('mvn -B -s settingsFileId -fae clean install')
     }
+
+    def "[maven.groovy] update pom element"() {
+        setup:
+            def pomURL = "https://raw.githubusercontent.com/kiegroup/kogito-runtimes/master/pom.xml"
```

Comment:
we should place every resource we need in `resources` folder. So please add this pom.xml file to `/test/resources` folder and load it as
```
def pomFile = getClass().getResource('/pom.xml')
def pom = new File(pomFile.toURI()).text
```